### PR TITLE
add serializer file

### PR DESCRIPTION
**Pull request by kentcdodds**
**What**: This adds a root-level `serializer.js` file

**Why**: It will make it much easier to use this package for the common use-case. People will be able to configure the serializer using the [`snapshotSerializers`](https://jestjs.io/docs/en/configuration.html#snapshotserializers-array-string) config:

```json
{
  "snapshotSerializers": [
    "jest-emotion/serializer"
  ]
}
```

**How**: Created a new file called `serializer.js` in the root of the project that simply creates the serializer as documented.

**Checklist**:

- [x] Documentation
- [ ] Tests N/A
- [x] Code complete

### CODE REVIEW

File: packages/jest-emotion/serializer.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const { createSerializer } = require('./dist')
```
Comment (mitchellhamilton) [inline packages/jest-emotion/serializer.js]: Could you change this to requiring `jest-emotion` instead of `./dist`?

Comment (kentcdodds) [reply packages/jest-emotion/serializer.js]: Interesting! Will that work!?

**Comment by tkh44** [comment]
Thanks @kentcdodds

File: packages/jest-emotion/serializer.js (hunk 1 above)
Comment (mitchellhamilton) [reply packages/jest-emotion/serializer.js]: Yeah, node should look in the node_modules folder, go into the jest-emotion folder and look at the main property in the package.json.

**Comment by ndelangen** [comment]
Awesome! just what I needed!

**Pull request merged**